code/xcvm/cosmwasm/contracts/interpreter/src/contract.rs

```diff
@@ -323,14 +318,9 @@ pub fn interpret_spawn(
 
 	let mut response = Response::default();
 	for (asset_id, balance) in assets.0 {
-		if balance.amount.is_zero() {
-			// We ignore zero amounts
-			continue
-		}
```

Comment:
Why not keep it?

Reply:
waste of time and resources and code space. i am fine sending zero, also it is per spec.

Reply:
any comments my it is needed? i do not see value, only waste